src/components/layout/layout.js

```diff
@@ -66,50 +76,47 @@ const CFDText = styled(Text)`
     @media ${device.bp1060} {
         font-size: 14px;
     }
-
     @media ${device.tablet} {
         font-size: 12px;
     }
-
     @media ${device.mobileL} {
         font-size: 10px;
     }
 `
 
 export const CFDWarning = ({ is_ppc }) => {
     const { is_eu_country } = React.useContext(DerivStore)
-    return is_ppc || is_eu_country ? (
-        <CFDWrapper>
-            <CFDContainer>
-                <CFDText>
-                    <Localize
-                        translate_text="CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage. <0>68% of retail investor accounts lose money when trading CFDs with this provider.</0> You should consider whether you understand how CFDs work and whether you can afford to take the high risk of losing your money."
-                        components={[<strong key={0} />]}
-                    />
-                </CFDText>
-            </CFDContainer>
-        </CFDWrapper>
-    ) : (
-        <></>
-    )
+    if (is_ppc || is_eu_country) {
+        return (
+            <CFDWrapper>
+                <CFDContainer>
+                    <CFDText>
+                        <Localize
+                            translate_text="CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage. <0>68% of retail investor accounts lose money when trading CFDs with this provider.</0> You should consider whether you understand how CFDs work and whether you can afford to take the high risk of losing your money."
+                            components={[<strong key={0} />]}
+                        />
+                    </CFDText>
+                </CFDContainer>
+            </CFDWrapper>
+        )
+    }
+    return <></>
 }
 
 const Main = styled.main`
     margin-top: ${(props) =>
         props.use_eu_margin
-            ? (props.margin_top && `${props.margin_top + cfd_warning_height.desktop}rem`) ||
-              `${7 + cfd_warning_height.desktop}rem`
-            : (props.margin_top && `${props.margin_top}rem`) || `7rem`};
+            ? (props.margin_top && `${props.margin_top}rem`) || '7rem'
+            : (props.margin_top && `${props.margin_top}rem`) || '7rem'};
```

Comment:
What's the difference here yeah? Both conditions look the same. If so, we can just use the `&&` operator.

Reply:
Missed out on this 😅. unused rule when the warning is on top